AmplifyPlugins/API/AWSAPICategoryPlugin/Support/Utils/GraphQLRequest+toListQuery.swift

```diff
@@ -0,0 +1,33 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Amplify
+import AWSPluginsCore
+
+extension GraphQLRequest {
+
+    /// Retrieve a GraphQL List operation request
+    static func listQuery<ResponseType: Decodable>(responseType: ResponseType.Type,
```

Comment:
this method was refactored and moved over here from the Provider/List implementation 

previous conversation: https://github.com/aws-amplify/amplify-ios/pull/990/files#r548597032